Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/CapacityRequirement.mo

```diff
@@ -0,0 +1,177 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Subsequences;
+block CapacityRequirement
+  "Cooling capacity requirement"
```

Comment:
In section 5.2.4.7, it states that "...a 5-minutes rooling average of instantaneous values sampled at a minimum of every 30 seconds." 
Should this sampling be implemented in this sequence?

Reply:
yes, it's already in there, as a `movMea` block.

Reply:
I see. I think the inputs need to be values that are sampled at at least the 30s rate. If we would sample them anywhere inside of this sequence, that would not guarantee that the quoted statement is satisfied.